savefile.c

```diff
@@ -405,7 +406,11 @@ pcap_offline_read(pcap_t *p, int cnt, pcap_handler callback, u_char *user)
 		}
 
 		if ((fcode = p->fcode.bf_insns) == NULL ||
-		    bpf_filter(fcode, data, h.len, h.caplen)) {
+		    (filter_out = bpf_filter(fcode, data, h.len, h.caplen))) {
+		   /* Google-specific fix for:
```

Comment:
Google-specific fix ???

Reply:
In the context of the bug this probably means "nobody bothered to use the exact return value of this function until it broke some Google code, which I had to fix".

I wonder if it would be right to assign `h.caplen = -1` when `p->fcode.bf_insns == NULL`, because that's one of the effects of this change.

Reply:
Perhaps...
Without context, this comment is unnecessary and should be deleted.